.github/workflows/deploy-frontend-pr-previews.yml

```
@@ -812,3 +818,159 @@ jobs:
+            --allow-unauthenticated
+
+      - name: Get preview service url
+        id: preview-url
+        run: |
+          read service_url < <(gcloud run services describe ${{ needs.branch-name.outputs.lowercase }}-reports-preview \
+            --format='value(status.url)' \
+            --platform managed \
+            --region ${{ secrets.REGION }})
+          echo "url=${service_url}" >>$GITHUB_OUTPUT
+
+  reports-pr-comment:
```

actionlint (1.7.4)

key "reports-pr-comment" is duplicated in "jobs" section. previously defined at line:806,col:3. note that this key is case insensitive

(syntax-check)